### DISCUSSION TITLE
Multi-factor at system level (2FA, TOTP, MFA)

### DISCUSSION THREAD

**Discussion by lazynooblet** [rustdesk/rustdesk#6835] [General]
Reposting #3212 into feature-requests

I'd really like to see 2FA as a per-system configuration. I don't agree with the comment posted on #1031 saying it wouldn't make sense to have it at system level. There are some systems I want to secure access to using 2FA. Anydesk have implemented this and moving to Rustdesk feels less secure relying solely on passwords.

**Comment by rustdesk** [comment]
You do not need to agree or not agree with https://github.com/rustdesk/rustdesk/issues/1031, different users have different requirements.

**Reply by rustdesk** [reply]
and stop producing duplicate things. https://github.com/rustdesk/rustdesk/discussions/3212

**Reply by lazynooblet** [reply]
I mentioned #3212 in this post. If "Ideas" is the same as "Feature Requests" then it seems redundant. I'll post on #3212 instead, thanks.
